### DISCUSSION TITLE
Excessive top and bottom magin for code in the list

### DISCUSSION THREAD

**Discussion by Jeevhi** [tailwindlabs/tailwindcss#12466] [Help] [answered]
I am new to tailwind and using it with markdown. When I use a list without any code in it, the top and bottom margin is 0.25 rem which is nice.

```
- item 1
- item 2
```

the corresponding CSS is 
```
.prose li {
    margin-top: 0.25rem;
    margin-bottom: 0.25rem;
}

```
However, if the list has code in it, the margin shoots up to 1.25rem.

```
- item 1 with `code`
- item 1 with `code`

```

the corresponding CSS is,

```
.prose :where(.prose > ul > li > *:first-child):not(:where([class~="not-prose"],[class~="not-prose"] *)) {
  margin-top: 1.25em;
}
.prose :where(.prose > ul > li > *:last-child):not(:where([class~="not-prose"],[class~="not-prose"] *)) {
  margin-bottom: 1.25em;
}
```
how do I fix it, I would like the margin to be fixed.

**Comment by wongjn** [comment]
Consider [customizing the CSS from `@tailwindcss/typography`](https://tailwindcss.com/docs/typography-plugin#customizing-the-css) to suit your needs.

**Reply by Jeevhi** [reply]
I tried adding the following to the `tailwind.config.js` to test css but seems to be not working (though if I only use li, it works). Can you please elaborate?  


the following does not work
```
'ul li code': {
                backgroundColor: '#fff',
 },
```

though the following code works but applies to the whole li which I don't want
```
'li': {
                backgroundColor: '#fff',
 },
```

I used backgroundColor instead of margin so that it's easy to visualize during testing

**Reply by wongjn** [reply, marked as answer]
Consider checking you have followed [the documentation for customizing the CSS of `@tailwindcss/typography`](https://tailwindcss.com/docs/typography-plugin#customizing-the-css). For example: https://play.tailwindcss.com/KLQiSsA5oG?file=config

**Reply by Jeevhi** [reply]
Thank you for your answer.  In my case, the following worked for me (with leading > ). I am making this as the answer 

```
> ul > li > code:  {
}

```
